### ISSUE TITLE
Export TrackParticles and Probes 3D to VTK in AM

### ISSUE THREAD

**Issue by Z10Frank** [SmileiPIC/Smilei#503]
## Description

It seems that exporting the particles gives their coordinates as in the AM reference frame, which is correct, but a 3D Probe in AM uses reference axes like a 3D simulation.
Bounds on the AM particles coordinates:
2.53892 to 243.062
-149.955 to 149.976
-149.809 to 149.616
Bounds on the 3D probe coordinates:
0 to 409.6
0 to 300
0 to 300

Note that the 3D Probes were defined on all the space according to the AM reference frame since they behave like particles:
```
DiagProbe(
    every = 3600,
    origin   = [0., -nr*dr,-nr*dr],
    corners  = [ [nx*dx,-nr*dr,-nr*dr], [0,nr*dr,-nr*dr],[0,-nr*dr,nr*dr] ],
    number   = [nx, 2*nr,2*nr],
    fields = list_fields
)
```
Is this the expected behaviour? Shouldn't their coordinates be centered on the `x` axis like for the particles? Their coordinates can be easily shifted with e.g. Paraview, just wondering if this is an inconsistency.

<img width="301" alt="Screenshot_2022-01-13_at_14 15 30" src="https://user-images.githubusercontent.com/9608804/153064025-f0fb676c-6807-40c7-a75c-f7f77b5c69ee.png">

**Comment by mccoys** [comment]
I just realized probes have a basic flaw when exporting to vtk. They are converted to a basic vtk 3d image with rectilinear axes. However, they should appear as a more complex skewed grid.

I will first make a basic workaround.

**Comment by Z10Frank** [comment]
Is it possible that the same flaw was present also when exporting AM Fields to VTK?

**Comment by mccoys** [comment]
No. Fields are fundamentally rectilinear and evenly spaced. In AM, the option build3d ensures the same geometry

**Comment by mccoys** [comment]
Fixed in 83dbb9f29a10af72
